gnovm/pkg/gnolang/nodes.go

```diff
@@ -1093,13 +1093,21 @@ func PackageNameFromFileBody(name, body string) Name {
 	return Name(astFile.Name.Name)
 }
 
-// NOTE: panics if package name is invalid.
-func ReadMemPackage(dir string, pkgPath string) *std.MemPackage {
+// ReadMemPackage initializes a new MemPackage by reading the OS directory
+// at dir, and saving it with the given pkgPath (import path).
+// The resulting MemPackage will contain the names and content of all *.gno files,
+// and additionally README.md, LICENSE, and gno.mod.
```

Comment:
All .md files should be stored.

Reply:
Sounds out of scope for this PR. Want me to create an issue?

Reply:
yes please